xikolo-ios/Base.lproj/Localizable.strings

```diff
@@ -26,7 +26,7 @@
 "email" = "Email";
 "password" = "Password";
 "register" = "Register";
-
+"you_are_enrolled" = "You are enrolled";
```

Comment:
Please just directly use the english string in Interface Builder, you can just use that as identification for translation as well